styles.css

```diff
@@ -0,0 +1,155 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+/* --Reset Styles-- */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* Styles for mobile screens */
+
+@media only screen and (min-width: 374px) and (max-width: 768px) {
+  .nav {
+    position: absolute;
+    width: 375px;
+    height: 92px;
+    left: 0;
+    top: 0;
+  }
+
+  .nav-bar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 378px;
```

Comment:
- [ ] Great job so far. Design is on point. Please remember, your site should be responsive up to the beginning of the desktop version(768). Please remove the steady widths you have set and set them to 100% so make them responsive. Please don't use the with to centralize items, you can display them flex, arrange and align them to where you want. Nice job.

![Screenshot (1003)_LI](https://user-images.githubusercontent.com/65298048/234885461-0b07a00e-9bf6-4baf-9cf5-4f058beb873d.jpg)

Reply:
I have finished fixing the required and recommended changes but, just for clarification, the only thing that I have to set to 100% is all the widths since I already declared the min and max widths in the @media using pixels?